packages/server-wallet/src/protocols/actions.ts

```diff
@@ -7,9 +7,9 @@ import {Bytes32} from '../type-aliases';
 Actions that protocols can declare.
 */
 
+export type Notice = Omit<Notification, 'jsonrpc'>;
```

Comment:
What's the point of the omission?

Reply:
The server wallet isn't really a json-rpc server, so I'd flip the question around on its head: what's the point of including it? 😛

Reply:
Makes sense, we should probably export & import something other than `Notification`